lib/climbing-crag-forecast.js

```diff
@@ -0,0 +1,6 @@
+class ClimbingCragForecast {

```

Comment:
Estáis usando docstrings en otra clase, pero en esta no. Aunque no se pide hasta más adelante (nivel 4), sería conveniente que fuerais consistentes.